BookApi/Data/BookDbContext.cs

```diff
@@ -0,0 +1,62 @@
+﻿using BookApi.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace BookApi.Data
+{
+    public class BookDbContext : DbContext
+    {
+        internal DbSet<Book> Books { get; set; } = null!;
+        internal DbSet<Category> Categories { get; set; } = null!;
+        internal DbSet<Publisher> Publishers { get; set; } = null!;
+        internal DbSet<Feedback> Feedbacks { get; set; } = null!;
+        internal DbSet<Author> Authors { get; set; } = null!; 
```

Comment:
In the future, it would be better to divide all tables in such way, that each database has only one table. So, when an error occurs, with any of the databases, the rest of them would work despite the broken one, as a principle of microservices.